src/summary/history/allergy-card-level-two.component.tsx

```diff
@@ -0,0 +1,175 @@
+                        <p>
+                          {allergy.resource.reaction[0].manifestation.map(
+                            manifestation => `${manifestation.text} `
+                          )}
+                        </p>
+                      </span>
+                    </p>
+                    <p>
+                      {dayjs(
+                        allergy.resource.extension[0].valueDateTime
+                      ).format("MMM-YYYY")}
+                    </p>
+                    <p>
+                      {dayjs(patientAllergy.meta.lastUpdated).format(
+                        "DD-MMM-YYYY"
+                      )}
+                    </p>
+                    <svg
+                      className={`omrs-icon ${styles.headerIcons}`}
+                      fill="rgba(0, 0, 0, 0.54)"
+                    >
+                      <use xlinkHref="#omrs-icon-chevron-right" />
+                    </svg>
+                  </div>
+                  <div className={styles.allergyComment}>
+                    <p></p>
+                    <p className={styles.allergyCommentText}>
+                      <p className="omrs-type-body-large">
+                        {allergy.resource.note && allergy.resource.note[0].text}
+                      </p>
+                      <p>more ...</p>
+                    </p>
+                  </div>
+                </div>
+              </>
+            );
+          })}
+      </SummaryCard>
+    );
+  }
+
+  function displayNoDrugAllergy() {
+    return (
+      <SummaryCard
+        name="Allergy"
+        match={props.match}
+        styles={{ width: "100%" }}
+      >
+        <SummaryCardRow>
+          <SummaryCardRowContent>
+            <HorizontalLabelValue
+              label="ALLERGEN"
+              labelStyles={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+              value="UPDATED"
+              valueStyles={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+            />
+          </SummaryCardRowContent>
+        </SummaryCardRow>
+
+        <SummaryCardRow>
+          <HorizontalLabelValue
+            label="Patient does not have known allergies"
+            labelClassName="omrs-bold"
+            value="June 2019"
+            valueStyles={{
+              color: "var(--omrs-color-ink-medium-contrast)",
+              marginRight: "1.625rem"
+            }}
+          />
+        </SummaryCardRow>
+      </SummaryCard>
+    );
+  }
+
+  function displayNoAllergenHistory() {
+    return (
+      <div>
+        <SummaryCard
+          name="Allergy"
+          match={props.match}
+          styles={{ width: "100%" }}
+        >
+          <div className={styles.allergyMargin}>
+            <p className="omrs-bold">
+              The patient's allergy history is not documented.
+            </p>
+            <p className="omrs-bold">
+              Please{" "}
+              <span className={`${styles.allergyLink} omrs-underline`}>
```

Comment:
This should be an `<a>` instead of a `<span>`, since it's a link. We can get rid of the `allergyLink` css class, since the designs in figma are just for a normal link.